src/main/java/com/novi/backend_spring_boot_tech_it_easy_controller/controllers/TelevisionsController.java

```diff
@@ -0,0 +1,38 @@
+package com.novi.backend_spring_boot_tech_it_easy_controller.controllers;
+
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+
+public class TelevisionsController {
+    @GetMapping("/televisions")
+    @ResponseBody
+    public String getAllTelevisions() {
+        return "List Televisions";
+    }
+
+    @GetMapping("/televisions/{model}")
+    @ResponseBody
+    public String getTelevision(@PathVariable String model) {
+        return "Type television" + model;
+    }
+
+    @PostMapping("/api/television")
+    public <TelevisionRequest> ResponseEntity<String> createTelevision(@RequestBody TelevisionRequest televisionRequest) {
+        String responseMessage = String.format("Television received: %s", televisionRequest.getClass());
+
+        return new ResponseEntity<>(responseMessage, HttpStatus.CREATED);
+    }
+
+
+    public <TelevisionRequest> ResponseEntity<String> updateTelevision(@PathVariable("id") Long id,
```

Comment:
Deze methode mist nog een `@PutMapping("televisions/{id}")`. Je vraagt in je parameters om een PathVariable, maar die is er nu niet.

Het idee van een PutMapping is dat je een televisie met een specifiek id gaat ophalen en gaat aanpassen met de gegevens die in de RequestBody staan. Dus in tegenstelling tot de PostMapping, maak je geen nieuw object, maar pas je een bestaand object aan.